intro-to-python/environment-setup/README.md

```diff
@@ -0,0 +1,33 @@
+# Python Environment
+
+_Ada Build - Intro to Python - Lesson 0_
+
+# Learning Goals
+
+* Learn to use Python in the [Google Colab] to complete Ada Build lessons.
+* Understand how [repl.it](https://repl.it/) will be use.
+
+## Video Intro
+
+The [Introduction to Google Colab video]() will serve as an introduction to the programming environment that you will use for the subsequent lessons. Please view this X minute video before continuing.
+
+## Google Colab
+
+For the Learning to Code portion of the Ada Build Curriculum you will review notes and complete exercises in a series of lessons in [Google Colab](./intro-to-python/00_preparing_to_code.ipynb). Colab is google's hosting of Jupyter Notebook, an open-source web application that allows you to create and share documents that contain live code, equations, visualizations and narrative text. Uses for Jupyter Notebook include: data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more. Here we are using colab and jupyter notebook to create an interative intro to python course.
+
```

Comment:
Just a note that I added this here because when I logged in with my non-ada github account and non-ada Google account I was confronted with this.

![authorize with github](https://github.com/Ada-Developers-Academy/ada-build/blob/build-reorg/intro-to-python/environment-setup/images/authorize-with-github.png?raw=true)